src/transcript/transcript.controller.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  HttpStatus,
+  Param,
+  Post,
+} from '@nestjs/common';
+import { TranscriptService } from './transcript.service';
+import { CreateTranscriptDto, UpdateTranscriptDto } from './dto/transcript.dto';
+import { Transcript } from './entity/transcript.model';
+
+@Controller('transcripts')
+export class TranscriptController {
+  constructor(private readonly transcriptService: TranscriptService) {}
+
+  // @Post('file')

```

Comment:
please decide what to do with this code, we cant leave commented code, remove it or add it in the form you wanted it